### PR TITLE
Various GNOME leaf package updates

### PR DESCRIPTION
**Pull request by jtojnar**
- [x] Built and run the packages on NixOS (sandbox)
- [x] Verified that Déjà dup can backup to Google Drive.

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -958,6 +958,8 @@ in {
 
   pydocumentdb = callPackage ../development/python-modules/pydocumentdb { };
 
+  PyDrive = callPackage ../development/python-modules/PyDrive { };
```
Comment (worldofpeace) [inline pkgs/top-level/python-packages.nix]:
```suggestion
  pydrive = callPackage ../development/python-modules/pydrive { };
```

I believe attributes in `python-packages.nix` should be lowercase.
Think we want to normalize `pname` that way also.

Comment (jtojnar) [reply pkgs/top-level/python-packages.nix]: Do you have a reference for that? I only see [15.17.1.2.1. Packaging a library](https://nixos.org/nixpkgs/manual/#packaging-a-library), which says “One of the arguments is the name of the package, which consists of a basename (generally following the name on PyPi)”

Comment (worldofpeace) [reply pkgs/top-level/python-packages.nix]: https://nixos.org/nixpkgs/manual/#contributing `attribute names` and the one about the `name` attribute is much harder to fine. (or maybe the latter isn't documented yet).

Comment (jtojnar) [reply pkgs/top-level/python-packages.nix]: Done thanks.

**Comment by jonringer** [comment]
if the tests are missing from pypi, i would just `fetchFromGitHub`

**Comment by jtojnar** [comment]
IIRC they were also broken in 0.6.0 on Git (but working on master), so I decided to disable them instead, rather than switching to unstable.

File: pkgs/desktops/gnome-3/core/gnome-user-docs/default.nix [hunk 2]
```diff
@@ -1,26 +1,42 @@
...
+    libxml2
+    yelp-tools
+  ];
+
+  enableParallelBuilding = true;
+
   passthru = {
-    updateScript = gnome3.updateScript { packageName = "gnome-user-docs"; attrPath = "gnome3.gnome-user-docs"; };
+    updateScript = gnome3.updateScript {
+      packageName = "gnome-user-docs";
```
Comment (worldofpeace) [inline pkgs/desktops/gnome-3/core/gnome-user-docs/default.nix]: pname?

File: pkgs/development/python-modules/BabelGladeExtractor/default.nix [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, lib
+, buildPythonPackage
+, fetchPypi
+, Babel
+, lxml
+}:
+
+buildPythonPackage rec {
+  pname = "BabelGladeExtractor";
```
Comment (worldofpeace) [inline pkgs/development/python-modules/BabelGladeExtractor/default.nix]: :slightly_frowning_face: aww, forgot to mention to rename this one as well.

**Review by worldofpeace — CHANGES_REQUESTED** [review]

File: pkgs/desktops/gnome-3/apps/gnome-documents/default.nix [hunk 4]
```diff
@@ -1,36 +1,86 @@
...
+    gtk3
+    glib
+    gsettings-desktop-schemas
+    gdk-pixbuf
+    gnome3.adwaita-icon-theme
+    evince
+    libsoup
+    webkitgtk
+    gjs
+    gobject-introspection
```
Comment (worldofpeace) [inline pkgs/desktops/gnome-3/apps/gnome-documents/default.nix]: gobject-introspection in buildInputs?

**Review by worldofpeace — APPROVED** [review]
Otherwise, LGTM. Feel free to merge after the previous comment is addressed.

File: pkgs/desktops/gnome-3/apps/gnome-documents/default.nix (hunk 4 above)
Comment (jtojnar) [reply pkgs/desktops/gnome-3/apps/gnome-documents/default.nix]: libgdprivate links against it.

**Pull request merged**